src/utils/test/validAttr.test.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import validAttr from '../validAttr'
+import validAttr from '@emotion/is-prop-valid'
 
 describe('validAttr', () => {
```

Comment:
Let’s maybe remove these tests.

I assume that the package on its own is sufficiently tested

Reply:
I waffled back and forth on that. In case we switch again in the future, I was thinking we should keep these. They did reveal some missing props 😄

Reply:
Actually our list might not be tested, gonna shamelessly copy ur tests later to cover that ;)

Reply:
Actually our list might not be tested, gonna shamelessly copy ur tests later to cover that ;)